frontend/src/utils/api.ts

```diff
@@ -269,7 +269,7 @@ window.api = {
           searchPath: searchParams.join('_').toLowerCase(),
           repoName: `${repo.displayName}`,
           location: `${repo.location.location}`,
-          checkedFileTypeContribution: 0,
+          checkedFileTypeContribution: -1,
```

Comment:
Any reason we are using `-1` instead of `undefined` here as the default value? I feel that `undefined` or maybe `null` would be more intuitive for developers.

Reply:
The initial reason was so I did not want to deal with `undefined`, however I do not have a strong opinion on this. Let me update it to `undefined`.